03week/mastermind.js

```diff
@@ -75,6 +75,7 @@ function mastermind(guess) {
   //else test entry is bad, alert user to try again
 
   if (valid(guess)) {
+    board[board.length] = `${guess} : ${generateHint(guess)}`;
     if (guess === solution) {
       console.log('You Win!')
```

Comment:
Please `return` statements in your `mastermind` function instead of console logging.